scripts/test_runner/setup_runner_host.sh

```diff
@@ -0,0 +1,44 @@
+# MIT License
```

Comment:
this file's permissions disallow execution

Reply:
permission added